crates/core/tedge_actors/src/servers/mod.rs

```diff
@@ -151,6 +156,9 @@ pub trait Service<Request: Message, Response: Message>:
     /// The client provides a `response_sender` on which it wants to response to be sent to.
     /// In exchange the client is returned a request sender on which its requests will have to be sent.
     fn connect_client(&mut self, response_sender: DynSender<Response>) -> DynSender<Request>;
+
+    /// Create a simple message box connected to a server box under construction.
+    fn new_client_box(&mut self) -> SimpleMessageBox<Response, Request>;
```

Comment:
Wondering if this function should this be hidden behind a test feature flag to make the intent clear?

Reply:
Hidden behind `#[cfg(feature = "test-helpers")]`